backend/platal/auth/tests.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+from django.test import TestCase
+
+from platal.auth import factories
+
+
+class FactoryTest(TestCase):
+
+    def test_account_factory(self):
+        f = factories.AccountFactory()
+        f.save()
```

Comment:
Since its a DjangoModelFactory, the default strategy is "create": the object is already saved when the factory returns.